train/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# tensorflow == 2.12.1 # TODO why not tensorflow < 2.13 as in eynollah/requirements.txt
```

Comment:
This TODO can be removed, `train` requires the same TF version as the main tool (`< 2.13`) rather than the specific pinned version.

Reply:
[8a9b4f8](https://github.com/qurator-spk/eynollah/pull/187/commits/8a9b4f8f55de9a2e51fd72af1894e771fe44f348)